README.md

```diff
@@ -14,8 +14,8 @@
     <h1>📗 Table of Contents</h1>
   </summary>
 
-- [Space Travelers' Hub](#space-travelers-hub)
-- [📖 Space Travelers' Hub ](#-space-travelers-hub-)
+- [Check Weather](#Check-Weather)
+- [📖 Check Weather ](#-Check-weather-)
   - [🛠 Built With ](#-built-with-)
     - [Tech Stack ](#tech-stack-)
     - [Key Features ](#key-features-)
```

Comment:
- [ ] Dear @binyamolango, Please keep in mind, The README file is expected to contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Please add the acknowledgment.

Reply:
- Well resolved 👍🏽